db/queries.py

```diff
@@ -3,44 +3,63 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return Chat.objects.filter(title__startswith="Untitled")
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    return list(Message.objects.filter(
+        sent__year="2015").values_list("user__first_name",
+                                       "user__last_name"))
 
 
 def get_actual_chats() -> list[Chat]:
-    pass
+    return Chat.objects.filter(
+        message__sent__year__gt="2020")
 
 
 def get_messages_contain_authors_first_name() -> list[Message]:
-    pass
+    return Message.objects.filter(
+        text__contains=F("user__first_name"))
 
 
 def get_users_who_sent_messages_starts_with_m_or_a() -> list[User]:
-    pass
+    return User.objects.filter(
+        Q(message__text__startswith="a") | Q(
+            message__text__startswith="m"))
```

Comment:
Look at the example, and make the same 
![image](https://user-images.githubusercontent.com/104218550/204483017-7fc36a8b-0271-4d23-b110-056384893e71.png)

In other similar cases too